src/config.py

```diff
@@ -3,10 +3,21 @@
 
 @dataclass
 class Config:
+    # General config
     batch_size: int = 64
     workers: int = 1
     dataset_dir: str = '../datasets/zalando-hd-resized'
     dataset_mode: str = 'test'
+    num_epochs: int = 5
+    learning_rate: float = 0.0002
+    num_classes: int = 10
+    num_input_channels: int = 1
+    num_val_samples: int = 25
+
+    # For development allow setting number of batches to not run the whole dataset
+    max_batches: int = 0
```

Comment:
I have added this flag so we can set a limited number of batches because if not its impossible to develop since each execution takes ages to load the whole dataset.